### PR TITLE
created the template for user profile edit form

### PR DESCRIPTION
**Pull request by franciscobenedict**
[this task](https://uktrade.atlassian.net/browse/ED-178)

- Created the template view for the edit user profile page
- setup the sass file for styling

screen shot 2016-10-14 at 16 37 58
screen shot 2016-10-14 at 16 38 16
screen shot 2016-10-14 at 16 37 29

### CODE REVIEW

File: registration/views.py [hunk 1]
```diff
@@ -1,123 +0,0 @@
...
-
-    def done(self, *args, **kwargs):
-
-        data = forms.serialize_company_profile_forms(self.get_all_cleaned_data())
-        response = api_client.company.update_profile(data)
-        if response.status_code == http.client.OK:
-            response = redirect('company-detail')
-        else:
-            response = TemplateResponse(self.request, self.failure_template)
-        return response
```
Comment (richtier) [inline registration/views.py]: oops. we want this file.

Comment (richtier) [inline registration/views.py]: oops. we want this file.

**Comment by konradko** [comment]
@franciscobenedict please fix this PR

**Comment by konradko** [comment]
@franciscobenedict When merging this PR please use GitHubs "Squash and merge" and remove the commit messages not related to the change - no need to pollute the commit history.

File: registration/views.py [hunk 2]
```diff
@@ -45,6 +45,7 @@ class RegistrationView(SessionWizardView):
 
     def get_template_names(self):
         return [
+            'user-profile-edit-form.html',
```
Comment (richtier) [inline registration/views.py]: we don't want this change. do:
```sh
git checkout origin/master -- registration/views.py
git commit --amend
git push origin user-profile-edit-form -f
```

**Comment by franciscobenedict** [comment]
@richtier can you further review this please? Template and scss files are now in and conflicts fixed.

**Pull request merged**